src/app/app.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+import { todo } from './todo'
+import { todoservice } from './todo.service';
+
+@Component({
+  selector:'todo',
+  template : `
+    <h1>Todo</h1>
+    <p></p>
+    <ul>
+      <li *ngFor= "let w of mywork">
```

Comment:
use specific variable name always. not w, a, d, test, arr etc.

Reply:
ok sir!